app/airq/commands/base.py

```diff
@@ -32,8 +32,13 @@ def handle(self) -> typing.List[str]:
 
     def _get_menu(self) -> typing.List[str]:
         return [
-            "Reply R for places with better air, "
+            "Reply R for safer places, "
             "D for details, "
             "L for previous zip, "
-            "and ? for information on hazebot."
+            "and ? for hazebot info."
+        ]
+
+    def _get_about(self) -> typing.List[str]:
```

Comment:
Given this is only use in the `ShowAbout` command, can we put this code directly in `ShowAbout.handle`?